rfcs/src/intro.md

```diff
@@ -0,0 +1,53 @@
+# Introduction
+
+Kani is an open-source verification tool that uses automated reasoning to analyze Rust programs. In order to 
+integrate feedback from developers and users on future changes to Kani, we decided to follow a light-weight 
+"RFC" (request for comments) process.
+
+## When to create an RFC
+
+As a rule of thumb you should create an RFC if you are making changes that impact the user experience. New 
+APIs, new features, as well as depreciation, should follow an RFC process. We also ask for substantial architectural 
```

Comment:
IMO, we should be explicit about a lighter-weight process option here. Changes are usually going to be:

1. Simple enough to not need an RFC
2. Complex enough to merit discussion and get consensus on design choices, but perhaps as a github issue rather than an RFC proper.
3. Important enough to merit an RFC-like process, especially when it comes to soliciting customer feedback.

Reply:
I would prefer start with 2 modes only. For the case where it merit discussion and get consensus, I would prefer using an RFC PR instead of an issue. It's OK if the RFC document is fairly small.

We can adjust as we go.